packages/tc-api-rest-handlers/lib/neo4j-query-builder.js

```diff
@@ -141,8 +143,20 @@ const queryBuilder = (method, input, body = {}) => {
 			queryParts.push(...addRelationshipQueries);
 			updateParameter(addRelationshipParams);
 		}
+		const isRelationship = identifyRelationships(type);
+		const initialContentRels = Object.keys(
+			initialContent,
+		).filter(propName => isRelationship(propName));
+
 		context.willAddRelationships = !!Object.keys(addedRelationships).length;
+		context.willCreateRelationships = !_isEmpty(
```

Comment:
In what circumstances does this return something different to willAddRelationships?

Reply:
`willAddRelationships ` includes all the relationship changes, creating relationships or updating relationships' properties of relationships. I couldn't think better name for property. I thought it might be better to change `willAddRelationships` to `willModifyRelationships` but the name already exists.